pyproject.toml

```diff
@@ -216,17 +216,30 @@ max-complexity = 25
 [tool.ruff.lint.per-file-ignores]
 "**/{doc}/*" = ["D103"]
 "**/{examples}/*" = ["B018", "D103", "D400", "E402"]
-"**/{tests}/**/test_*" = ["D100", "D103", "E402"]
+"**/{tests}/**/test_*" = ["D100", "E402"]
 "__init__.py" = ["D104", "E402"]
+"nilearn/_utils/*" = ["B028"]  # 7 errors to fix
+"nilearn/connectome/*" = ["B028"]  # 5 errors to fix
+"nilearn/connectome/**/{tests}/*" = ["D103"]  # 35 errors to fix
```

Comment:
@bthirion 

fixing error D103 in the test_*.py files should ensure all tests have a docstring

the comments here give an estimate of many tests need to be 'fixed' in each subpackage